tests/drivers/candev/main.c

```diff
@@ -196,6 +198,10 @@ int main(void)
     isrpipe_init(&rxbuf, (uint8_t *)rx_ringbuf, sizeof(rx_ringbuf));
 #if IS_USED(MODULE_PERIPH_CAN)
     puts("Initializing CAN periph device");
+#if defined(MCU_SAMD5X)
+    /* Standby pin of the CAN transceiver. Must be low */
+    gpio_init(GPIO_PIN(PC, 13), GPIO_IN);
```

Comment:
This sounds very board specific and should be handled by the board / the `periph_candev` driver with a PIN defined in `board.h`

Reply:
I would say it's a board specific because this pin is related to the CAN transceiver integrated in the board. I'll switch it there